src/apriltags_driver.cpp

```diff
@@ -15,7 +15,11 @@
 #include <lcm/lcm-cpp.hpp>
 #include "LcmWrapper.hpp"
 
+#ifdef BOT_CORE_H
+#include <lcmtypes/bot_core.h>
+#else
```

Comment:
I don't understand this.  `bot_core` is widely used on many projects, why is this change necessary all of a sudden?  Is it possibly a bug in bot_core's cmake code?